internal/dao/log_item.go

```diff
@@ -89,9 +92,25 @@ func (l *LogItem) Render(paint string, showTime bool, bb *bytes.Buffer) {
 		bb.WriteString("[-::] ")
 	}
 
-	if index > 0 {
-		bb.Write(l.Bytes[index+1:])
-	} else {
-		bb.Write(l.Bytes)
+	bb.Write(colorizeJSON(l.Bytes[index+1:], showJson))
+}
+
+func colorizeJSON(text []byte, showJson bool) []byte {
+	if !showJson {
+		return text
+	}
+	var obj map[string]interface{}
```

Comment:
What if you have int or object as an index, not a string?

Reply:
Json object keys must always be strings: https://www.w3schools.com/js/js_json_objects.asp#:~:text=Keys%20must%20be%20strings%2C%20and,number

Reply:
It's true if your application returns an object. But it can return an array, right?

Reply:
But nvm, I'm going to resolve this. Thanks!